esp/src/src/nls/es/hpcc.ts

```diff
@@ -295,6 +297,7 @@ export = {
     FirstN: "PrimerosN",
     FirstName: "Primer Nombre",
     FirstNRows: "Primeras N Filas",
+    FirstNSortBy: "Ordenar Por",
```

Comment:
Based on the name of the message, this should be Ordenar por las primeras N filas"